app/modules/abtest/featured-video-render-order-helper.js

```diff
@@ -0,0 +1,147 @@
+import DomHelper from '../dom-helper';
+import Ads from '../ads';
+import JWPlayerVideoAds from '../video-players/jwplayer-video-ads';
+import {track} from '../../utils/track';
+
+export function initializeMobileWiki() {
+	if (!window.mobileWikiInitialized) {
+		window.mobileWikiInitialized = true;
+		if (typeof FastBoot === 'undefined' && M.getFromShoebox('serverError')) {
+			// No need to load Ember in browser on server error page
+			return;
+		}
+		M.loadScript('/mobile-wiki/assets/vendor.js', false, false, 'anonymous');
+		M.loadScript('/mobile-wiki/assets/mobile-wiki.js', false, false, 'anonymous');
+	}
+}
+
+export function setFeaturedVideoPlayer(player) {
+	window.featuredVideoPlayer = player;
+}
+
+function createCookie(name, value, days = 14) {
+	let expires = "";
+	if (days) {
+		var date = new Date();
+		date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
+		expires = "; expires=" + date.toUTCString();
+	}
+	document.cookie = `${name}=${value}${expires}; path=/; domain=${M.getFromShoebox('runtimeConfig.cookieDomain')}`;
+}
+
+function onCreate(bidParams, player) {
+	const adsInstance = Ads.getInstance();
+	if (adsInstance.jwPlayerAds && adsInstance.jwPlayerMoat) {
+		adsInstance.jwPlayerAds(player, bidParams);
+		adsInstance.jwPlayerMoat(player);
+	}
+
+	player.on('autoplayToggle', ({enabled}) => {
+		createCookie('featuredVideoAutoplay', (enabled ? '1' : '0'));
+	});
+
+	player.on('captionsSelected', ({selectedLang}) => {
+		createCookie('featuredVideoCaptions', selectedLang);
+	});
+
+	player.once('adImpression', () => {
+		initializeMobileWiki();
+	});
+
+	player.once('videoStart', () => {
+		initializeMobileWiki();
+	});
+
+	// initialize mobile wiki after 8 seconds if it hasn't been initialized yet
+	setTimeout(() => {
```

Comment:
If we initialize mobile-wiki on setupError of JW we should have everything covered, right? Then we could drop this fallback

Reply:
I'd leave it for unknown scenarios ;)